src/main/java/at/hannibal2/skyhanni/features/commands/TransferCooldown.kt

```diff
@@ -0,0 +1,59 @@
+package at.hannibal2.skyhanni.features.commands
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.LorenzWorldChangeEvent
+import at.hannibal2.skyhanni.events.MessageSendToServerEvent
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.seconds
+
+object TransferCooldown {
+
+    private val config get() = SkyHanniMod.feature.misc.commands
+    var isActive: Boolean = false
+    private var action: (() -> Unit)? = null
+
+    @SubscribeEvent
+    fun onWorldLoad(event: LorenzWorldChangeEvent) {
+        if (!config.transferCooldown || isActive) return
+        isActive = true
+        DelayedRun.runDelayed(3.seconds) {
```

Comment:
remove `isActive` and store the `SimpleTimeMark` that `runDelayed` returns because if an error occurs by running the action `isActive` will stay true which will mean none of the commands will ever go through until a world swap